html/15-file-upload/index.html

```diff
@@ -21,6 +20,17 @@
         <input type="file" accept=".pdf">
     </label>
     <br>
+    <label for="picture">Take a picture of your face:
+        <input type="file" id="picture" accept="image/*" capture="user">
```

Comment:
@MiljkovicD Wenn du das Input in ein Label verschachtelst, brauchst du dann eine for-id Verknüpfung?